scripts/code_style.py

```diff
@@ -137,6 +139,20 @@ def get_src_files(since: Optional[str]) -> List[str]:
                          is_file_autogenerated(filename))]
     return src_files
 
+def get_submodule_hash(commit: str, submodule: str) -> str:
```

Comment:
Pylint says: please document this function.

Though as a human, I think its purpose is clear enough. But a one-line description wouldn't hurt.

One thing that is not clear without looking at the code is the behavior if the submodule doesn't exist in the commit. For this particular script, we don't care (see my comment below), but if the function is reused in the future, we might care.